src/components/FeatureFlag.tsx

```diff
@@ -26,9 +52,18 @@ const FeatureFlag = (
 
     let isEnabled = defaultValue;
 
-    // set the flag value if found and check if inverted
     if (flag !== undefined) {
-        isEnabled = invert ? !flag.enabled : flag.enabled;
+
+        if (flag.enabled && flag.strategies) {
+            // check if strategies match
+            const flagStrategyResult = checkFlagStrategies(flag, flagsClient ? flagsClient.getUserId() : undefined);
+            isEnabled = flagStrategyResult;
+        } else {
+            // if the flag is disabled or no strategies are present then result is false
+            isEnabled = false;
```

Comment:
When no strategy is present we should always fall back to the default strategy. It is the default.